_posts/2015-05-29-chrome-push-api.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title: W3C Push API Crash Course
+description: Getting started with push notifications in Google Chrome
+tags: [technology, push, web, w3c, javascript]
+author: nkcmr
+---
+W3C’s [Push API](http://www.w3.org/TR/push-api/) is exciting. Almost as exciting as all the possibilities that arise from having a persistent presence in your users browser. At Zumba, we already have ideas for it, and I was in charge of doing the gritty work of getting notifications to show up in the browser, but also developing a way to manage who gets what notifications. I had a lot of questions along the way, and I will lay them out and give you straight answers. Let's get pushing already!
```

Comment:
* "users" `->` "users'"
* "who gets what notifications" `->` "who gets those notifications"
* Run-on sentence (fixed): "I had a lot of questions along the way that I will lay out and give you straight answers."